.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+rvm: 2.3.1
+notifications:
+ email: false
+cache:
+- bundler
+- apt
+addons:
+  postgresql: '9.4'
+before_install:
+- sudo apt-get update -qq
+- sudo apt-get install -qq postgresql-server-dev-9.4 libpq-dev
+before_script: rails db:migrate
```

Comment:
yeah we need to create. @luciademoja  replace the current command with `bundle exec rake db:setup`